### DISCUSSION TITLE
save source ip of syslogs

### DISCUSSION THREAD

**Discussion by q16marvin** [openobserve/openobserve#4972] [Q&A]
Hi,

first: realy cool tool, thank you very much. its realy is to setup and getting running.
for me i only need to collect syslog from about 5 pfsense server :)

so for now i could create different streams, one stream for one pfsense server. 
on the end it would be awesome that openobserve will save also the source ip of any syslog row... is there any way?

Thx!

Erik

**Comment by prabhatsharma** [comment]
Unfortunately syslog does not offer the functionality of capturing that.

**Reply by q16marvin** [reply]
oh okay thats sad... but could currently i set from which ip's clients are able to send syslogs to openobserve... could openobserve not "tag" every log from which ip it comming?

**Reply by prabhatsharma** [reply]
Instead of sending logs directly to OpenObserve you could send it to fluentbit which can capture the IP and then forward it to OpenObserve. Check - https://github.com/fluent/fluent-bit/issues/1413
